File: exercicios_c/secao6/exercicio2.c
//Elabore um algoritmo que leia um número, se positivo, armazene-o em a,
//se negativo, em b. No final, mostrar o resultado.

#include<stdio.h>

int main(void){
    int n, a, b;

    printf("Digite um numero: ");
    scanf("%d", &n);

    if (n>0){
        a = n;
        printf("O resultado é %d, numero positivo", a);
    }else if (n<0){
        b = n;
        printf("%d, numero negativo", b);
    }else{
        printf("Valor neutro");
    };
}

File: exercicios_py/secao6/exercicio2.py
#Elabore um algoritmo que leia um número, 
# se positivo, armazene-o em a,
# se negativo, em b. 
# No final, mostrar o resultado.

n = int(input("Digite um número: "))
if n > 0:
    a = n
    print("Número positivo, n = a")
elif n < 0:
    b = n
    print('Número negativo, n = b')
else:
    print("Número neutro")